Readability.js

```diff
@@ -1112,7 +1112,7 @@ Readability.prototype = {
           });
 
           // But first check if we actually have something
-          if (this._attempts[0].textLength === 0) {
+          if (this._attempts[0].textLength) {
```

Comment:
You missed out the `!`, which unfortunately is quite important...

Reply:
(I would just fix this myself and land manually, but I'm afraid my github fu is really weak and I'm in the middle of something else right now)